### ISSUE TITLE
Binary class segmentation

### ISSUE THREAD

**Issue by jackyko1991** [MiguelMonteiro/VNet-Tensorflow#1]
In the readme file you write that:

```
In a binary segmentation problem you could use num_classes=1 with a sigmoid loss and in a three class problem you could use num_classes=3 with a softmax loss.
```

why not using num_classes = 2 with softmax loss? What is the superiority of sigmoid loss?

**Comment by MiguelMonteiro** [comment]
Not much, it's the same but it uses a bit less memory.
